core/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .utils import *
 from .datetimes.shared import *
+from django.db.models import Q
```

Comment:
I have to add this line because without it /api/graphql does't work correctly. Without this line graphQL UI doesn't show. And there is error "Could not import 'openIMIS.schema.schema' for Graphene setting 'SCHEMA'. ImportError: cannot import name 'Q' from 'core' (/home/user/IMIS/openimis-be-core_py/core/__init__.py)." 
This situation happens when we have both "openimis-be_py" and "openimis-be-core_py" on "develop" branch on local environment.